interface/patient_file/summary/stats_full.php

```diff
@@ -181,6 +181,8 @@ function newEncounter() {
                 <th scope="col"><?php echo xlt('Occurrence'); ?></th>
                 <?php if ($focustype == "allergy") { ?>
                   <th scope="col"><?php echo xlt('Reaction'); ?></th>
+                <?php }
+                if ($focustype == "allergy" || $focustype == "medical_problem") { ?>
```

Comment:
note this didn't work well previously (it didn't work for edit ui of items), so instead leveraged the javascript method:
https://github.com/openemr/openemr/commit/3e1e77b1b0e1f87d030a607713974e4d8c6aaa70

Also, the medical_problem should use it's own list, which will default to Confirmed

Reply:
Did you mean Default in Database or Default in UI?

Reply:
Also, this file will not be used in edit UI so will this change have any effect here?

Reply:
Regarding default, meant in the UI.

You are correct that I was mistaken on this file (assumed it was the add edit script), so disregard that part of the comment :)